test/test_backend/routes/tst_graph.py

```diff
@@ -2,26 +2,32 @@
 
 How to run:
     python -m unittest discover
+
+todo: when finished developing these tests, revert to actual test
```

Comment:
## Wrap-up todo's
A reminder of some quick things to do to standardize this test file once we are done developing it.